### PR TITLE
Added tests to physics.vector for increased coverage

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed
Some testing

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/vector/tests/test_frame.py [hunk 1]
```diff
@@ -288,3 +290,48 @@ def test_issue_11498():
...
+                                              ['a', 'b', 'c'], [0, 1]))
+    raises(TypeError, lambda: ReferenceFrame('N', ['a', 'b', 'c'],
+                                             ['a', 'b', 'c'], [0, 1, 2]))
+    N = ReferenceFrame('N')
+    assert N[0] == CoordinateSym('N_x', N, 0)
+    assert N[1] == CoordinateSym('N_y', N, 1)
+    assert N[2] == CoordinateSym('N_z', N, 2)
+    raises(ValueError, lambda: N[3])
+    N = ReferenceFrame('N', ['a', 'b', 'c'])
+    assert N['a'] == N['a']
```
Comment (smichr) [inline sympy/physics/vector/tests/test_frame.py]: These look like "do-nothings"...what is being tested?

Comment (oscargus) [reply sympy/physics/vector/tests/test_frame.py]: Agreed. There is some code in `__getitem__` that is being explicitly tested. But I have now figured out a better way to do it. (I was a bit surprised that it just returned the same things as was put in... Asking for N['a'] does some matching and then returns N.x, which is N['a']...))

**Comment by smichr** [comment]
The TODO at the top of the printing test file can be removed:
```
formerly, when the TODO was written things looked like this

-    expected = (u'\u03b1*\x1b[94m\x1b[1mn_x\x1b[0;0m\x1b[0;0m + '
-                u'sin(\u03c9)*\x1b[94m\x1b[1mn_y\x1b[0;0m\x1b[0;0m + '
-                u'\u03b1\u22c5\u03b2*\x1b[94m\x1b[1mn_z\x1b[0;0m\x1b[0;0m')
-
-    assert expected == pp.doprint(w)
-    assert expected == w._pretty().render()

and now they look like this

+    expected = """\
+ 2
+a  n_x + b n_y + c*sin(alpha) n_z\
+"""
+    uexpected = u("""\
+ 2
+a  n_x + b n_y + c⋅sin(α) n_z\
+""")
+
+    assert ascii_vpretty(v) == expected
```

Also, the newly added `strlines` will handle the raw lines escaping for you:
```python
>>> print(strlines(xx2._latex()))
(
r'\mathbf{\hat{n}_x}\otimes \mathbf{\hat{n}_y} + \mathbf{\hat{n'
r'}_x}\otimes \mathbf{\hat{n}_z}'
)
```

**Comment by oscargus** [comment]
I noted that you removed the TODO in #16322 

Yes, I could. I think about it, but then I prefer to break in "sensible" places so that one can read the output more easily... (I've been considering making a similar function handling tags for MathML... For LaTeX it seems a bit trickier, maybe always break before the last { or \ on the row...)

**Comment by oscargus** [comment]
Since this does not change anything and there has been a review (on the first part) of it, I merge this.

**Pull request merged**